### ISSUE TITLE
No proxy settings?

### ISSUE THREAD

**Issue by gmarthews** [haecker-felix/gradio#119]
It would be nice to be able to change the proxy settings so that I didn't need to change the network settings.

**Comment by gmarthews** [comment]
That said i changed the network settings and it still cannot connect so maybe this is a bug.

**Comment by haecker-felix** [comment]
@gmarthews yeah currently gradio doesn't take the proxy informations from the system. Currently it isn't possible to get gradio using a proxy.

I think the best way is, to read the system proxy settings.

**Comment by geneliu** [comment]
If we support proxy from system proxy settings (like chrome browser) we need make sure we support dynamic settings something like http://proxy.lbs.example.com/proxy.pac
